File: 2025-03-28/Title_Helper.java
public class Title_Helper {
    public static String manipulateString(String input) {
        String[] words = input.trim().split("\\s+");
        StringBuilder result = new StringBuilder();
        for (int i = 0; i < words.length; i++) {
            String word = words[i];
            if (i > 0) {
                result.append("_");
            }
            result.append(Character.toUpperCase(word.charAt(0)))
                    .append(word.substring(1));
        }        return result.toString();
    }
    public static void main(String[] args) {
        String input = "Find Closest Number to Zero\n";
        String result = manipulateString(input);
        System.out.println(result);
    }
}

File: 2025-09-25/Title_helper.py
class TitleHelper:
    @staticmethod
    def manipulate_string(input_str: str) -> str:
        words = input_str.strip().split()
        result = []
        for i, word in enumerate(words):
            formatted_word = word[0].upper() + word[1:] if word else ""
            result.append(formatted_word)
        return "_".join(result)


if __name__ == "__main__":
    input_str = input()
    result = TitleHelper.manipulate_string(input_str)
    print(result)
